templates/CRM/Admin/Form/MessageTemplates.tpl

```diff
@@ -165,8 +165,8 @@
       });
       function showHideUpload(type) {
         var show = (type == 1) ? false : true;
-        $("#msg_html").toggle(show);
-        $("#msg_text, #pdf_format").toggle(show);
+        $("#msg_html_section").toggle(show);
+        $("#msg_text_section, #pdf_format").toggle(show);
```

Comment:
These 2 lines could be combined into 1, just adding an extra comma to the jQuery selector.